src/NaluParsing.C

```diff
@@ -781,6 +781,21 @@ namespace YAML
       wallData.z0_ =
           node["roughness_height"].as<sierra::nalu::RoughnessHeight>();
     }
+    if (node["RANS_abl_bc"])
+    {
+      wallData.RANSAblBcApproach_ =
+          node["RANS_abl_bc"].as<bool>();
+    }
+    if (node["fixed_velocity"])
+    {
+      wallData.uFixed_ =
+          node["fixed_velocity"].as<double>();
+    }
+    if (node["fixed_height"])
+    {
+      wallData.zFixed_ =
+          node["fixed_height"].as<double>();
+    }
```

Comment:
If user activates RANS ABL boundary condition, we should make uRef and zRef parameters mandatory. 

```suggestion
    if (node["RANS_abl_bc"]) {
        const bool is_activated = node["RANS_abl_bc"].as<bool>();
        wallData.RANSAblBcApproach_ = is_activated;

        if (is_activated) {
            wallData.uFixed_ = node["fixed_velocity"].as<double>();
            wallData.zFixed_ = node["fixed_height"].as<double>();
        }
    }
```